test/e2e/shell.go

```diff
@@ -56,8 +55,13 @@ var _ = Describe("Shell", func() {
 	}
 })
 
-func absOrDie(path string) string {
-	out, err := filepath.Abs(path)
+// Returns the root directory of the Kubernetes source tree, assuming the test lives inside test/e2e.
```

Comment:
Actually it's based on the current working directory, not the location of the test.